tests/integration/tables/last.cpp

```diff
@@ -27,14 +27,9 @@ TEST_F(last, test_sanity) {
   // 2. Check size before validation
   ASSERT_FALSE(data.empty());
   // 3. Build validation map
-  ValidationMap row_map = {
-      {"username", NormalType}
-      {"tty", NormalType}
-      {"pid", NonNegativeInt}
-      {"type", IntMinMaxCheck(7, 8)}
-      {"time", NonNegativeInt}
-      {"host", verifyEmptyStringOrIpAddress}
-  };
+  ValidationMap row_map = {{"username", NormalType} {"tty", NormalType} {
+      "pid", NonNegativeInt} {"type", IntMinMaxCheck(7, 8)} {
+      "time", NonNegativeInt} {"host", verifyEmptyStringOrIpAddress}};
```

Comment:
This won't compile, each element (for instance `{"username", NormalType}`) is missing a comma at the end, that's why the weird formatting from clang-format.

Reply:
Ah damn, just to confirm, it should be:
```
ValidationMap row_map = {
    {"username", NormalType},
    {"tty", NormalType},
    {"pid", NonNegativeInt},
    {"type", IntMinMaxCheck(7, 8)},
    {"time", NonNegativeInt},
    {"host", verifyEmptyStringOrIpAddress},
};
```
Right?

Reply:
yes!